events/message.js

```diff
@@ -7,30 +9,62 @@ const profanityActions = require('../eventActions/profanityActions');
 const citadelActions = require('../eventActions/citadelActions');
 const accountabilityActions = require('../eventActions/accountabilityActions');
 const chainMessageAction = require('../eventActions/checkChainMessage');
-const highlightActions = require ('../eventActions/highlightActions');
+const highlightActions = require('../eventActions/highlightActions');
 const afkAction = require('../eventActions/afkMessageCheckAction');
-const gratitudeActions =  require('../eventActions/gratitudeActions');
-
+const gratitudeActions = require('../eventActions/gratitudeActions');
 module.exports = async (client, message) => {
-
 	if (!message.guild || message.author.bot) return;
+	const bannedWords = await db.get(`bannedWords-${message.guild.id}`);
+	if (db.get(`bannedWords-${message.guild.id}`) == null) {
+		db.set(`bannedWords-${message.guild.id}`, ['pussy']);
+	}
 	const args = message.content.split(/\s+/g); // Return the message content and split the prefix.
 	const command =
-		message.content.startsWith(config.prefix) &&
-		args.shift().slice(config.prefix.length).toLowerCase();
+    message.content.startsWith(config.prefix) &&
+    args.shift().slice(config.prefix.length).toLowerCase();
 
 	if (command) {
 		const commandfile =
-			client.commands.get(command) ||
-			client.commands.get(client.aliases.get(command));
-
+      client.commands.get(command) ||
+      client.commands.get(client.aliases.get(command));
 		if (commandfile) {
 			commandfile.execute(client, message, args).then(() => {
 				message.delete(1500);
 			}); // Execute found command
 		}
+		if (!commandfile) {
+			if (bannedWords.some((word) => message.content.includes(word))) {
```

Comment:
Banned words check here should go in `eventActions/profanityActions.js`, not `message.js`

Reply:
ok!